addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/ZoneHandler.java

```diff
@@ -23,6 +23,7 @@
 
 public class ZoneHandler extends AbstractOmnilinkHandler {
     private Logger logger = LoggerFactory.getLogger(ZoneHandler.class);
+    private ZoneStatus zoneStatus;
```

Comment:
its been a while since I have used the volatile keywords in Java, out of curiosity, whats your reasoning for using it here?